marconi-chain-index/src/Marconi/ChainIndex/Orphans.hs

```diff
@@ -102,10 +104,10 @@ instance Serialise C.ScriptData where
   decode = fromCBOR
 
 instance FromJSON C.ScriptData where
-    parseJSON (Aeson.String s) =
-        either (const mempty) pure
-            $ C.deserialiseFromCBOR C.AsScriptData (Text.encodeUtf8 s)
-    parseJSON _ = mempty
```

Comment:
Using this, the RPC test consistently fails if scriptData FromJSON

Reply:
https://github.com/input-output-hk/plutus-apps/pull/984 was just merged. Does the fix in the ScriptData ToJSON/FromJSON solve the issue?

Reply:
Yes, as I had the exact change to fix the issue.